alert-database/src/main/resources/liquibase/6.13.0/job-execution-status.xml

```diff
@@ -4,10 +4,10 @@
     <changeSet author="psantos" id="create-job-execution-status-table">
```

Comment:
likely a very minor detail, but this ID could be amended to be "create-job-completion-status-table"
```suggestion
    <changeSet author="psantos" id="create-job-completion-status-table">
```